class ClientErrorException(Exception):
    pass 

class ValueErrorException(Exception):
    pass